test/verify/check-users

```diff
@@ -31,7 +31,12 @@ def performUserAction(browser, user, action):
     browser.click(f"#accounts-list tbody tr:contains({user}) .pf-c-dropdown__menu li:contains({action})")
 
 
-def createUser(browser, user_name, real_name, password, locked, force_password_change, custom_home_dir=None, verify_created=True):
+def createUser(browser, machine, user_name, real_name, password, locked, force_password_change, custom_home_dir=None, default_shell=None, custom_shell=None, verify_created=True):
+    if default_shell is None:
+        if "debian" in machine.image or "ubuntu" in machine.image:
+            default_shell = "/bin/sh"
+        else:
+            default_shell = "/bin/bash"
```

Comment:
This is weird. The default shell ought to be /bin/bash everywhere. /bin/sh is the default shell for scripts, but it should *never* be the default login shell for users. dash is really not meant for humans, it's awful.

Reply:
On our debian-stable image, the "useradd -D" gives me default as "SHELL=/bin/sh"

Reply:
Yes, but on Debian `useradd` is really the wrong tool to use, like this it creates accounts with no usable shell. `adduser` is the canonical tool on Debians, or at least useradd with an explicit --shell.